### ISSUE TITLE
[UI / UX] Remove individual filters

### ISSUE THREAD

**Issue by gabitoesmiapodo** [gnosis/conditional-tokens-explorer#463]
Once a filter is applied, there's no way to remove / reset individual filters (you can reset all filters by hiding the column).

We need a design and implementation for this.

Also https://github.com/gnosis/conditional-tokens-explorer/issues/531#issuecomment-718620039

**(update in comments)**

**Comment by gabitoesmiapodo** [comment]
Design here:

https://zpl.io/VDLZ6lr (notice that the link only activates when there's something to remove)
https://zpl.io/bLeQgwm
https://zpl.io/bAL6zWx
https://zpl.io/b6rN5Ry ("Clear" link font is a bit bigger here)

**Some notes about this:**

- I did some research on some major sites about their filter usage and their "clear filter" controls. UX / UI seems to be inconsistent at best, but usually there's no "Clear" button for dropdowns so I'm not adding them. It seems obvious that the user can select the "All" option and it seems easy enough for them to do it IMO. Anyway, we could add them later if we need to.
- Strangely enough, some sites like Amazon have no filter removing feature.
- Some sites put filter "capsules" at the top (eBay) which can be used to remove individual filters,, but that's probably not the best option here so I went with simple "link type" buttons.
